common-pg/src/main/scala/com/socrata/pg/soql/SqlFunctions.scala

```diff
@@ -71,8 +71,8 @@ object SqlFunctions extends SqlFunctionsLocation with SqlFunctionsGeometry with
       paramPosition = Some(Seq(0,0))),
     SignedMagnitudeLinear ->
       formatCall(
-        "case when %s = 1 then floor(%s) else sign(%s) * floor(abs(%s)/%s + 1) end",
-        paramPosition = Some(Seq(1,0,0,0,1))),
+        "case when %s = 1 then floor(%s) else floor(%s/%s) end",
```

Comment:
Actually the `%s = 1` case is spurious. This entire statement can simply be `floor(%s/%s)`.

There was one hypothesis that the original code was protecting against division by 0 by doing a `+ 1` but that's not the case, bc it would've had to be `floor(abs(%s) / (%s + 1))`